README.md

```diff
@@ -176,7 +176,13 @@ $ y: echo -e "$((x+10))\n$((x+20))"
 
 ### Variable options
 
-For lines starting with `$` you can add extra options using `---`.
+For lines starting with `$` you can add use`---` to parse parameters to `fzf`.
+* `--allow-extra` : handles `fzf` option `--print-query`. `enter` will prefer a selection,
+    `tab` will prefer the query typed.
+* `--multi` : forwarded option to `fzf`.
+* `--header` | `--headers` | `--header-lines` : forwarded option to `fzf`
```

Comment:
Header and headers are for backwards compatibility only. Theyre not intended to be used

Reply:
Ok so only `--header-lines` stays.